models/Table/ExhibitPage.php

```diff
@@ -258,4 +258,14 @@ protected function filterByItem($select, $item_id)
         );
         $select->where('exhibit_block_attachments.item_id = ?', $item_id);
     }
+
```

Comment:
This looks like it goes along with the other search stuff that doesn't seem to be used